website/index.html

```diff
@@ -5052,7 +5066,7 @@ <h2>----------- RESULTS BELOW -----------</h2>
         "multi_target_mode", "essence_of_the_red", "periodic_damage", "can_trigger_enrage",
         "ability_queue", "first_hit_heroic_strike", "use_slam", "use_sl_in_exec_phase", "use_ms_in_exec_phase", "use_mortal_strike",
         "use_sweeping_strikes", "dont_use_hm_when_ss", "fungal_bloom", "full_polarity", "battle_squawk", "ferocious_inspiration",
-        "drums_of_battle", "haste_potion", "bloodlust", "insane_strength_potion", "heroic_potion"];
+        "drums_of_battle", "haste_potion", "bloodlust", "insane_strength_potion", "heroic_potion","use_devastate","use_devastate_in_exec_phase"];
```

Comment:
nit: ... you already know ;)

Reply:
Don't know how I missed the first comment here. Between this project and something I'm working on at work I get yelled at so much about my inconsistent use of spaces that I'm basically having nightmares about missed spaces.